sdk/storage/storage-file-datalake/src/sas/DataLakeSASSignatureValues.ts

```diff
@@ -86,11 +87,63 @@ export interface DataLakeSASSignatureValues {
    */
   pathName?: string;
 
+  /**
+   * Optional. Beginning in version 2020-02-10, this value defines whether or not the {@link pathName} is a directory.
+   * If this value is set to true, the Path is a Directory for a Directory SAS. If set to false or default, the Path
+   * is a File Path for a File Path SAS.
+   *
+   * @type {boolean}
+   * @memberof DataLakeSASSignatureValues
+   */
+  isDirectory?: boolean;
+
+  /**
+   * Optional. Beginning in version 2020-02-10, indicate the depth of the directory
+   * specified in the canonicalizedresource field of the string-to-sign. The depth of the directory is the number of directories
```

Comment:
>number of directories [](start = 107, length = 21)

Is this the number of level of directories, or number of directories?

Reply:
Copied the comments from their documentations. Can't think of a better description.